fastapi/routing.py

```diff
@@ -151,6 +155,32 @@ async def run_endpoint_function(
         return await run_in_threadpool(dependant.call, **values)
 
 
+async def handle_endpoint(
+    request: Request,
+    dependant: Dependant,
+    body: Optional[Union[Dict[str, Any], FormData]],
+    dependency_overrides_provider: Optional[Any],
+    is_coroutine: bool,
+) -> Tuple[Any, Optional[BackgroundTasks], Response]:
+
+    solved_result = await solve_dependencies(
+        request=request,
+        dependant=dependant,
+        body=body,
+        dependency_overrides_provider=dependency_overrides_provider,
+    )
+    values, errors, background_tasks, sub_response, _ = solved_result
+    if errors:
+        raise RequestValidationError(errors, body=body)
+    else:
```

Comment:
Remove else statement here. No need for that. It just causes decreasing indentation level.

Reply:
fixed